src/joint_embedding/methods/jae/train/script.py

```diff
@@ -55,7 +56,7 @@
     mod1_reducer.fit(mod1_data)
```

Comment:
In PCA transformation, the input should be raw count input_mod.layers['count'] rather than input_mod.X (please refer to the file 1_pca_pretrain.py, line)
So the line 52, and line 64 here should be changed to 
`mod1_data = scale * normalize(input_mod1.layers["counts"],norm='l1', axis=1)`, mod2_data = scale * normalize(input_mod2.layers["counts"],norm='l1', axis=1)

Reply:
:+1: Good catch!